toolkit/templates/forms/textbox.html

```diff
@@ -64,24 +64,28 @@
     {% if large %}
       {% if max_length_in_words %}
         <div class="word-count">
-          <textarea
-            class="text-box text-box-large{% if error %}-with-error{% endif %}"
-            name="{{ name }}"
-            id="input-{{ name }}"
-            data-max-length-in-words="{{ max_length_in_words }}"
-            {% if question_advice or hint %}aria-describedby="{{ [question_advice_id, answer_advice_id]|join(' ')|trim }}"{% endif %}
-          >
+          <div>
```

Comment:
Why is the extra div required?

Reply:
https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/471/files#diff-6a148e617c7e875e3f54f660a13ae54dR13

https://github.com/CezaryDanielNowak/css-resize-polyfill/blob/master/demo.html#L37

Reply:
It operates on (applies classes to) an empty parent div

Reply:
It doesn't work on the existing parent div?

Reply:
Yeah, but it felt a bit dangerous doubling up, don't want to override anything

Reply:
I'm just a bit worried about how multiple divs might interfere with a screenreader; it would be good to test this branch in the empathy lab